Source/AwsCommonRuntimeKit/event-stream/EventStreamMessage.swift

```diff
@@ -54,36 +54,37 @@ extension EventStreamMessage {
         }
         let addCHeader: () throws -> Int32 = {
             return try header.name.withCString { headerName in
+                let headerNameLength = UInt8(header.name.utf8.count)
```

Comment:
I wonder if `.utf8.count` is really just O(N) under the hood...

If it is, I'd be tempted to just call `strlen(cstring)` because it seems harder to mess up, compared to checking that we always do `swiftstring.utf8.count` and never accidentally do `swiftstring.count`

really wish `withCString()` passed both pointer and length to the closure

Reply:
Updated and removed withCString. Yeah, I agree it's really easy to mess up, given that we need to pass `.utf8.count` to C APIs instead of `.count`.